examples/gallery/plot/legend.py

```diff
@@ -0,0 +1,26 @@
+"""
+Legend
+------
+
+The :meth:`pygmt.Figure.legend` method can automatically create a legend for
+symbols plotted using :meth:`pygmt.Figure.plot`.
+"""
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.basemap(projection="x1i", region="0/7/3/7", frame=True)
```

Comment:
I think it's nicer to use a list here
```suggestion
fig.basemap(projection="x1i", region=[0, 7, 3, 7], frame=True)
```